pack.yaml

```diff
@@ -6,7 +6,7 @@ keywords:
   - zabbix
   - monitoring
 version: 1.0.0
-author: Hiroyasu OHYAMA
-email: user.localhost2000@gmail.com
+author: Ian Price
+email: ianmprice@gmail.com
```

Comment:
I think the original pack authorship doesn't need to change. After all, you initially authored the pack and have a credit for that.

The other part is the list of active maintainers which worth updating in the https://github.com/StackStorm-Exchange/stackstorm-zabbix/blob/8a613dad10808cc5cd2f32e278e09d189b067cdf/.github/CODEOWNERS#L11

Reply:
Thank you very much for giving the precious advice. I got it 👍